slides/developer-guide/working-with-document/merge-powerpoint-presentations/_index.md

```diff
@@ -33,7 +34,10 @@ Not supported.
 
 #### **HTTP POST**
 
-##### **Example 1**
+{{< tabs tabTotal="2" tabID="1" tabName1="Example 1" tabName2="Example 2" >}}

```

Comment:
I don't like the idea of grouping examples in tabs, because there should be examples for different language, and it's languages that need to be organized in tabs.
You do not have to add examples for all the languages immediately, if you are not ready, but please keep in mind that some day they should appear.

Reply:
All examples in these sections have been written with .NET SDK.
A similar structure is used [here](https://docs.aspose.cloud/slides/download-a-slide-from-a-powerpoint-presentation/#http-get), for example.
You should suggest something instead.